mne/source_estimate.py

```diff
@@ -2246,7 +2246,7 @@ def __init__(self, data, vertices=None, tmin=None, tstep=None,
     def plot_surface(self, src, subject=None, surface='inflated', hemi='lh',
                      colormap='auto', time_label='time=%02.f ms',
                      smoothing_steps=10,
-                     transparent=None, alpha=1.0, time_viewer=False,
+                     transparent=None, alpha=1.0, time_viewer='auto',
```

Comment:
These will also need to be changed in `_3d.py`

Reply:
This was done in #7380 no?

Reply:
Ahh, yes I guess it was